pages/[...slug].vue

```diff
@@ -0,0 +1,54 @@
+<!--
+name: slug
+description: this a page for dynamic content, in Nuxt we can use dynamic routes to create pages that match a pattern.
+  this page will match any pattern as it has `...` in it. This is a catch-all routes and it used for `@nuxt/content`
+-->
+
+<script setup lang="ts">
+const route = useRoute()
+
+const { data: page } = await useAsyncData(route.path, () => queryContent(route.path).findOne())
+
+const minHeight = computed(() => {
+  // TODO: fix dehydration issue
+  // const footerHeight = document?.querySelector('footer')?.clientHeight ?? 0
+  return `calc(100vh - ${125 * 2}px)`
+},
+)
+
+if (page.value) {
+  const title = page.value?.title ?? 'Dev Edmonton Society'
+  const description = page.value.data?.description ?? 'A community of developers in Edmonton.'
```

Comment:
nit: description could be -  Fostering a stronger software development community in Edmonton. We are made up of people who believe that an impactful, diverse, and collaborative developer community enriches us all. We realize this vision by sharing knowledge and ideas, understanding the community and its needs, promoting events and initiatives, and supporting technology meetups and organizations.